CONTRIBUTING.md

```diff
@@ -17,41 +16,59 @@ This guide will help you get started and outline some things you should know whe
   (install [directly](https://www.ruby-lang.org/en/documentation/installation),
   or via a ruby manager like [rvm](https://rvm.io/)
   or [rbenv](https://github.com/sstephenson/rbenv))
-  * Language the tests are written in
+  * language the tests are written in
 * [ShellCheck](https://github.com/koalaman/shellcheck)
-  * Used in the linting process to find common errors in the Bash code
+  * used in the linting process to find common errors in the Bash code
 
 
 ## Setup
 
+* install the [requirements](#requirements)
 * fork and clone the repository to your machine
-* `bundle` to install ruby gems
+* run `bundle` to install ruby gems
+* run `bundle exec rake` to make sure all tests pass
```

Comment:
The setup is to allow for local development. I would say that making sure the tests work before doing changes is part of it. Similar to how making sure the tests are green is part of the setup of a feature branch.